packages/material-ui/src/AccordionSummary/AccordionSummary.js

```diff
@@ -169,21 +150,22 @@ AccordionSummary.propTypes = {
    */
   expandIcon: PropTypes.node,
   /**
-   * Props applied to the `IconButton` element wrapping the expand icon.
+   * This prop can help a person know which element has the keyboard focus.
+   * The class name will be applied when the element gain the focus through a keyboard interaction.
```

Comment:
```suggestion
   * The class name will be applied when the element gains the focus through keyboard interaction.
```

Reply:
I'm taking note to apply the change in the next batch for v5.